bcdevexchange/wwwroot/css/style.css

```diff
@@ -153,7 +153,7 @@ h3 {
 
 .main-header-area {
   background: #181818;
-  line-height: 3em;
+  line-height: 2.3em;

```

Comment:
Just confirming that this is supposed to be 2.3em and not 2em

Reply:
Good call on this one. 3em makes the the navbar too tall for my liking, so reducing to 2.3em helps. However, I've found that removing this style altogether has the same effect - so this line is unnecessary.